rackcity/api/serializers/asset_serializers.py

```diff
@@ -9,18 +9,22 @@
     NetworkPort,
     NetworkPortCP,
 )
+from . import SiteSerializer
 from .it_model_serializers import ITModelSerializer
 from .rack_serializers import RackSerializer
 from .change_plan_serializers import GetChangePlanSerializer
 from rackcity.api.serializers.fields import RCIntegerField
 import copy
 
+from ...models.model_utils import ModelType
```

Comment:
Can you change this to `rackcity.models.model_utils` pls - safer if stuff gets moved around

Reply:
pycharm did this 😠

Reply:
Gross!